python/ecl/grid/ecl_grid.py

```diff
@@ -1267,7 +1269,33 @@ def export_index(self, active_only = False):
         df = pandas.DataFrame(data=data, index=indx)
         return df
         
- 
+    def export_data(self, index_frame, kw):
+        if not isinstance(index_frame, pandas.DataFrame):
+            raise TypeError("index_frame must be pandas.DataFrame")
+        global_index = numpy.array( index_frame.index )
```

Comment:
This makes a new numpy array object with the global index - that should not be necessary.